tests/Integration/Behaviour/Features/Scenario/CatalogPriceRule/catalog_price_rules_listing.feature

```diff
@@ -0,0 +1,307 @@
+      | catalog price rule 3 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 4 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 5 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+    And I should be able to see following list of catalog price rules with language "en" with limit 50 offset 0 and total 1 and product "product2":
+      | name                 | currency   | country       | group    | from quantity  | reduction type| reduction value | shop     | includes tax| price | from                | to                  |
+      | catalog price rule 5 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+
+  Scenario: I can see correctly with attribute condition
+    When I add product product3 with following information:
+      | name[en-US] | Combination product  |
+      | type        | combinations         |
+    And I generate combinations for product product3 using following attributes:
+      | Color | [Red,Blue] |
+    Then I add following conditions to catalog price rule "catalogPriceRuleReference5":
+      | type           | value |
+      | attribute      | Blue  |
+    And I should be able to see following list of catalog price rules with language "en" with limit 50 offset 0 and total 4 and product "product1":
+      | name                 | currency   | country       | group    | from quantity  | reduction type| reduction value | shop     | includes tax| price | from                | to                  |
+      | catalog price rule 1 | usd        | UnitedStates  | visitor  | 1              | amount        | 111.50          | testShop | true        | 50    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 2 | usd        | UnitedStates  | visitor  | 1              | amount        | 50              | testShop | true        | 10    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 3 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 4 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+    And I should be able to see following list of catalog price rules with language "en" with limit 50 offset 0 and total 0 and product "product2":
```

Comment:
It's a bit weird to provide an empty table for this one, you should add a dedicated endpoint for this one like:

```suggestion
    And "product2" should have no catalog price rules with language "en"
```